src/components/BaseComponents/CheckAnswer/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useEffect, useState } from 'react';
+import type { PConnFieldProps } from '@pega/react-sdk-components/lib/types/PConnProps';
+import { useTranslation } from 'react-i18next';
+
+interface HmrcOdxTestProps extends PConnFieldProps {
+  // If any, enter additional props that only exist on this componentName
+  name?: string;
+  stepId?: any;
+}
+
+// Duplicated runtime code from React SDK
+
+// props passed in combination of props from property panel (config.json) and run time props from Constellation
+// any default values in config.pros should be set in defaultProps at bottom of this file
+export default function GDSCheckAnswers(props: HmrcOdxTestProps) {
+  const COMMA_DELIMITED_FIELD = 'CSV';
+  const { label, value, name, stepId, getPConnect } = props;
+  const [formattedValue, setFormattedValue] = useState<string | Array<string>>(value);
+  const pConn = getPConnect();
+  const actions = pConn.getActionsApi();
+  const containerItemID = pConn.getContextName();
+  const { t } = useTranslation();
+  let isCSV = false;
+
```

Comment:
The following might be a better approach. Again it's not an urgent update but correcting the useEffect dependencies is better. 
const isCSV = name?.includes(COMMA_DELIMITED_FIELD) && value?.includes(',');
  useEffect(() => {
    if (isCSV) {
      setFormattedValue(value.split(',').map((item: string) => item.trim()));
    }
  }, [value, isCSV]);